.changeset/fair-beans-shout.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Please add a roundtrip test. See https://github.com/finos/legend-studio/pull/358 for an example.